molo/core/templatetags/core_tags.py

```diff
@@ -679,6 +680,7 @@ def social_media_article(context):
         'twitter': twitter,
         'request': context['request'],
         'locale_code': locale,
+        'page': obj,
```

Comment:
We could use `return data.update({'page': obj}) if obj else data` to be sure that we're not breaking anything